x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/metrics/chart_header.tsx

```diff
@@ -23,7 +23,7 @@ export const ChartHeader = ({ title, metrics }: Props) => {
     <EuiFlexGroup gutterSize={'s'} responsive={false}>
       <EuiFlexItem grow={1}>
         <EuiText size={'s'}>
-          <h4>{title}</h4>
+          <h4 style={{ wordBreak: 'break-all' }}>{title}</h4>
```

Comment:
If the title is long is displayed like this:
<img width="400" alt="Screenshot 2021-04-14 at 12 53 02" src="https://user-images.githubusercontent.com/2249229/114700429-c9335c00-9d21-11eb-826a-0b9681ad94be.png">

Maybe we should cut it as we do in the tooltip?
<img width="200" alt="Screenshot 2021-04-14 at 13 00 02" src="https://user-images.githubusercontent.com/2249229/114700445-cdf81000-9d21-11eb-97a5-aafc3a73a8d6.png">